otelhelpers/simple_carrier_test.go

```diff
@@ -0,0 +1,60 @@
+package otelhelpers
+
+import (
+	"context"
+	"testing"
+
+	"go.opentelemetry.io/otel"
+)
+
+func TestSimpleCarrier(t *testing.T) {
+	carrier := SimpleCarrier{}
+	carrier.Clear() // clean up after other tests
+
+	// traceparent is the only key supported by SimpleCarrier
+	got := carrier.Get("traceparent")
+	if got != "" {
+		t.Errorf("got a non-empty traceparent value '%s' where empty string was expected", got)
+	}
+
+	carrier.Set("foobar", "baz")
+	if carrier.Get("foobar") != "baz" {
+		t.Error("did not get the expected value back in Set/Get test")
+	}
+
+	// traceparent is supported so this should work fine
+	tp := "00-b122b620341449410b9cd900c96d459d-aa21cda35388b694-01"
+	carrier.Set("traceparent", tp)
+
+	// even though 2 keys have been set at this point, the carrier only returns
+	// one key, traceparent
+	keys := carrier.Keys()
+	if len(keys) != 2 {
+		t.Errorf("expected exactly 2 keys from Keys() but instead got %q", keys)
```

Comment:
One or two?

Reply:
ohhh bad comment, will fix

Reply:
fixed in 9e17bec7c8659b6e5f6376b5bd27cd2f7e5a1d19